open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/VisualizationStyleOptionsProvider.java

```diff
@@ -135,4 +167,38 @@ public void actionPerformed(ActionEvent e)
         }
         return myResetAllStyleDataButton;
     }
+
+    /**
+     * Gets the `show toolbar labels` checkbox.
+     *
+     * @return the `show toolbar labels` checkbox
+     */
+    private JCheckBox getShowToolbarLabelsCheckBox()
+    {
+        if (myShowToolbarLabelsCheckBox == null)

```

Comment:
Personally I'm not a huge fan of lazy initializing UI elements as a matter of practice, even though it was done a lot in some of our code.  Not a big deal though, you can probably leave it unless you feel inclined to change it (i.e. remove the if check and change the method to "create" instead of "get").

Reply:
Good idea. Restructured class to initialize the panel when constructed.